vkubelet/helpers.go

```diff
@@ -0,0 +1,63 @@
+package vkubelet
+
+import (
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+)
+
+// populateSecretsAndConfigMapsInEnv populates Secrets and ConfigMap into environment variables
+func (s *Server) populateSecretsAndConfigMapsInEnv(pod *corev1.Pod) error {
```

Comment:
shall we change the function name to populateEnvironmentVariables as it supposes to handle the downward API too, in the future.